### ISSUE TITLE
Default Windows Volume Top Left Corner suddenly Visible

### ISSUE THREAD

**Issue by zragon** [irzyxa/Volume2#296]
Several days ago it works fine with default volume bar hidden, now, today, it does not hide the default volume bar even though i have uninstalled and tried other  versions (Volume 1.1.8.465)

![image](https://github.com/irzyxa/Volume2/assets/20996853/ca11fab3-632e-4671-9058-35fae62efb25)

**Desktop (please complete the following information):**
 - Windows version: Windows 10 Pro, 22H2  19045.3693
 - Volume² version: 1.1.7.449

**Comment by jimkimjat** [comment]
I had this problem too when I deleted all the keyboard hotkeys. What I did to fix it is:

1) Uninstall Volume2
2) Go to %appdata% or C:\Users\<user>\AppData\Roaming (it's to reset the settings, you will lose your configurations)
3) Delete the folder Volume2
4) Reinstall Volume2

It's a workaround until there's a real fix.

**Comment by irzyxa** [comment]
@jimkimjat you are correct. There must be 2 required actions in Hotkeys - VolumeUp and VolumeDown in this way we block default OSD.
